packages/providers/src/providers/unspeech/elevenlabs.ts

```diff
@@ -7,19 +7,89 @@ import { generateCRO } from '../../utils/generate-cro'
 
 /** @see {@link https://elevenlabs.io/docs/api-reference/text-to-speech/convert#request} */
 export interface UnElevenLabsOptions {
+  /**
+   * This parameter controls text normalization with three modes: ‘auto’, ‘on’, and ‘off’. When set to ‘auto’,
+   * the system will automatically decide whether to apply text normalization (e.g., spelling out numbers).
+   * With ‘on’, text normalization will always be applied, while with ‘off’, it will be skipped. Cannot be
+   * turned on for ‘eleven_turbo_v2_5’ model.
+   */
   applyTextNormalization?: 'auto' | 'off' | 'on'
+  /**
+   * Language code (ISO 639-1) used to enforce a language for the model. Currently only Turbo v2.5
+   * supports language enforcement. For other models, an error will be returned if language code is provided.
+   */
   languageCode?: string
-  model: 'eleven_monolingual_v2' | 'eleven_turbo_v2_5' | ({} & string)
+  /**
+   * Identifier of the model that will be used, you can query them using GET /v1/models.
+   * The model needs to have support for text to speech, you can check this using the
+   * can_do_text_to_speech property.
+   *
+   * Selection guide based on needs
+   *
+   * - Quality: Use `elevenlabs/eleven_multilingual_v2`
+   * - Low-latency: Use `elevenlabs/eleven_flash_v2` or `elevenlabs/eleven_flash_v2_5`
+   * - Multilingual support: Use `elevenlabs/eleven_multilingual_v2` or `elevenlabs/eleven_flash_v2_5`
+   *
+   * Selection guide based on use cases
+   *
+   * - Content Creation: Use `elevenlabs/eleven_multilingual_v2`
+   * - Conversational AI: Use `elevenlabs/eleven_flash_v2_5` or `elevenlabs/eleven_flash_v2`
+   * - Voice Changer (Speech to Speech): Use `elevenlabs/eleven_multilingual_sts_v2`
+   *
+   * @see {@link https://elevenlabs.io/docs/developer-guides/models}
+   */
+  model: 'elevenlabs/eleven_english_sts_v2' | 'elevenlabs/eleven_flash_v2' | 'elevenlabs/eleven_flash_v2_5' | 'elevenlabs/eleven_multilingual_sts_v2' | 'elevenlabs/eleven_multilingual_v2' | ({} & string)
```

Comment:
`model` should not contain a provider prefix

Reply:
This is a `unspeech` specific provider, so for pre-inlcluded Elevenlabs models, `elevenlabs/`prefix is required.